Configuration.props

```diff
@@ -12,11 +12,16 @@
       Condition=" Exists('$(MSBuildThisFileDirectory)Configuration.OperatingSystem.props') And '$(DoNotLoadOSProperties)' != 'True' "
   />
   <PropertyGroup>
+    <ConfigurationPropsImported>true</ConfigurationPropsImported>
```

Comment:
Question: can we rename `Configuration.props` to `Directory.Build.props`? https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019

Reply:
I started _roughly_ looking at this  in a separate branch in my fork - https://github.com/pjcollins/xamarin-android/tree/dirprops. I'd rather do this separately in the future if we do choose to migrate.